views/dpsSearchResults.njk

```diff
@@ -31,7 +31,7 @@
       <div class="govuk-grid-row">
         {% set rows = [] %}
         {% for u in results %}
-          {% set userLink = '<a href="' + maintainUrl + '/' + u.username + '/' + u.username + '/details" class="govuk-link" data-qa="edit-button-'
```

Comment:
for DPS users use the `username` as the `userId` since that is what prison api takes